preprod/digital-prison-reporting-lambdas.yml

```diff
@@ -2,7 +2,7 @@ release:
   project: digital-prison-reporting-lambdas
   release_type: gradle
   release_category: backend
-  tag: v0.0.12
+  tag: v0.0.14
```

Comment:
Where does this change release version 14?
https://github.com/ministryofjustice/modernisation-platform-environments/pull/9104/files#diff-8f71c80e7cab31ba347cb02e6fab2a75ea2dac81843b87546b021cf4fb96cfabR228

Reply:
Is this only for dev?

Reply:
The MP changes _use_ the artifact, but these changes are needed to _deploy_ the artifact to pre-prod and higher.